docs/manuals/source/DeveloperGuide/netprotocol.rst

```diff
@@ -245,3 +245,20 @@ The following diagram shows the activity on a Bareos TCP server (i.e. a |dir| or
   :Handle client message;
   end
 
+
+Debugging
+---------
+
+Developers who want to debug the message flow can use the network dump feature. This feature is only for specialists and is **not intended for use in a production system**.
```

Comment:
"This feature is only" twice (next sentence starts basically the same) and I'd use "experts" instead of "specialists", but that's just a matter of taste.

Reply:
Changed the text meaningful.